src/main/java/com/codeus/winter/config/DefaultBeanFactory.java

```diff
@@ -11,20 +12,35 @@
 import java.util.Map;
 
 /**
- * Default implementation of the {@link BeanFactory} interface
+ * Default implementation of the {@link BeanFactory} interface.
  */
 public class DefaultBeanFactory implements BeanFactory<Object> {
+    /**
+     * Storage for saving beans with Scope equal singleton.
+     */
     private final Map<String, Object> singletonBeans = new HashMap<>();
+    /**
+     * Storage for saving BeanDefinition.
+     */
     private final Map<String, BeanDefinition> beanDefinitions = new HashMap<>();
+    /**
+     * Storage for saving BeanPostProcessors.
+     */
     private final List<BeanPostProcessor> postProcessors = new ArrayList<>();
 
 
+    /**
+     * Request beans from the storage with specified parameters.
+     *
+     * @param name bean's name.
+     * @return bean object if its exist or else throw exception.
+     */
     @Nullable
     @Override
-    public Object getBean(String name) throws BeanNotFoundException {
-        var isPresent = singletonBeans.entrySet().stream()
-                .map(entry -> entry.getKey().equals(name))
-                .anyMatch(name::equals);
+    public Object getBean(@Nonnull final String name) throws BeanNotFoundException {
+        var isPresent = singletonBeans.keySet().stream()
+                .map(o -> o.equals(name))
+                .anyMatch(n -> n.equals(true));
 
         if (isPresent) {
```

Comment:
Could you please check if it's possible to use .ifPresentOrElse() here? This can make the code simple and more readable + remove redundant if-else